samcli/commands/check/bottle_necks.py

```diff
@@ -0,0 +1,75 @@
+"""
+Bottle neck questions are asked here. Data is saved in graph, but not calcualted here.
+"""
+import click
+
+from samcli.commands.check.resources.Graph import Graph
+from samcli.commands.check.resources.LambdaFunction import LambdaFunction
+from samcli.commands._utils.resources import AWS_LAMBDA_FUNCTION
+
+
+class BottleNecks:
+    def __init__(self, graph: Graph):
+        self._graph = graph
```

Comment:
Nice job on adding type hintings!
Can we also add type hintings for class members?
```
class BottleNecks:
    _graph: Graph                     # type hinting for self._graph
    _lambda_max_duration: int         # type hinting for self._lambda_max_duration
    def __init__(self, graph: Graph):
        self._graph = graph
```
Same goes for other classes.